app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetConfigurationFragment.kt

```diff
@@ -46,18 +46,23 @@ class UnreadWidgetConfigurationFragment : PreferenceFragmentCompat() {
         }
 
         unreadFolderEnabled = findPreference(PREFERENCE_UNREAD_FOLDER_ENABLED)!!
-        unreadFolderEnabled.onPreferenceChangeListener = Preference.OnPreferenceChangeListener { _, _ ->
-            unreadFolder.summary = getString(R.string.unread_widget_folder_summary)
-            selectedFolderId = null
-            true
+        if (selectedAccountUuid == null) {
```

Comment:
At the point where the fragment is created `selectedAccountUuid` is always null. So there's no need for this check.